src/fiatExchanges/utils.tsx

```diff
@@ -12,6 +10,23 @@ import Logger from 'src/utils/Logger'
 
 const TAG = 'fiatExchanges:utils'
 
+export enum FiatExchangeFlow {
```

Comment:
Good call on the nit.

Can you rephrase what you are suggesting with the second part? I don't understand. Its a bit tricky and i had trouble coming up with a good solution. The beginning of the flow can be CashIn/CashOut/Spend but the second half of the flow can only be CashIn/CashOut and i thought it would be cleaner for types if they were separate

Reply:
Oh and actually on the nit, i forgot I actually tried to do something like that and it doesn't work. Declaring something as a type has different behavior than declaring it as an enum and you can't do things like Unions with enums unfortunately